src/neo/SmartContract/NNS/DomainState.cs

```diff
@@ -0,0 +1,47 @@
+using Neo.IO;
+using Neo.SmartContract.Native.Tokens;
+using Neo.VM;
+using Neo.VM.Types;
+using System.IO;
+
+namespace Neo.SmartContract.NNS
+{
+    public class DomainState : Nep11TokenState
+    {
+        public UInt160 Operator { set; get; }
+        public uint TimeToLive { set; get; }
+        public string Name { set; get; }
+
+        public override int Size => UInt160.Length + sizeof(uint) + Name.GetVarSize();
+
+        public override void FromStackItem(StackItem stackItem){
+            Array @array = (Array)stackItem;
+            Operator = @array[0].IsNull ? null : @array[1].GetSpan().AsSerializable<UInt160>();
+            TimeToLive= (uint)@array[1].GetBigInteger();
+            Name = @array[2].IsNull ? null : System.Text.Encoding.UTF8.GetString(@array[2].GetSpan().ToArray());
```

Comment:
```suggestion
            Name = System.Text.Encoding.UTF8.GetString(@array[2].GetSpan().ToArray());
```

It's impossible be null.